src/CefBrowser/Runtime/WebBrowserClient.cs

```diff
@@ -419,15 +416,15 @@ private void HandleCefProcessLog(object sender, DataReceivedEventArgs e)
 
         #endregion
 
-        #region CEF Events
+        #region Browser Controlls
 
         /// <summary>
         ///     Sends a keyboard event to the CEF process
         /// </summary>
         /// <param name="keysDown"></param>
         /// <param name="keysUp"></param>
         /// <param name="chars"></param>
-        internal void SendKeyboardEvent(int[] keysDown, int[] keysUp, string chars)
+        public void SendKeyboardControlls(int[] keysDown, int[] keysUp, string chars)
```

Comment:
Controls, not Controlls :)
(Same with above)

Reply:
Bruh

Reply:
Well if you want what ur want from me, I except the best lol.

Reply:
yea yea yea, there you go.